 snapper (0.2.4-netgear3) unstable; urgency=low
 .
   * Allow snapshot access to everyone if "guest" is in the allowed users list.
Author: Justin Maggard <justin.maggard@netgear.com>

---

--- snapper-0.2.4.orig/server/Client.cc
+++ snapper-0.2.4/server/Client.cc
@@ -387,6 +387,10 @@ Client::check_permission(DBus::Connectio
     if (uid == 0)
 	return;
 
+    // Check if guest is allowed
+    if (contains(meta_snapper.uids, 99))
+	return;
+
     // Check if the uid of the dbus-user is included in the allowed uids.
     if (contains(meta_snapper.uids, uid))
 	return;
